marklogic-data-hub/src/test/java/com/marklogic/hub/security/DataHubOperatorTest.java

```diff
@@ -32,4 +40,25 @@ void readSchemas() {
                 "permissions on them. Since they may have no other read permissions, the operator requires this so that " +
                 "it can read entity models.");
     }
+
+    @Test
+    void testQBE(){
+        writeFinalJsonDoc("doc1.json", "{\"itemPrice\": 10}");
+        writeFinalJsonDoc("doc2.json", "{\"itemPrice\": 5}");
+
+        QueryManager queryMgr = runAsDataHubOperator().getFinalClient().newQueryManager();
+        String rawJSONQuery = "{ \"$query\": {\n" +
+            "    \"$and\": [\n" +
+            "        {\"itemPrice\" : { \"$ge\": 6 } },\n" +
+            "        {\"itemPrice\" : { \"$le\": 12 } } ],\n" +
+            "    \"$filtered\": true\n" +
+            "}}";
+        StringHandle rawHandle = new StringHandle(rawJSONQuery).withFormat(Format.JSON);
+        RawQueryByExampleDefinition querydef = queryMgr.newRawQueryByExampleDefinition(rawHandle);
+        SearchHandle resultsHandle = queryMgr.search(querydef, new SearchHandle());
+        assertEquals(1, resultsHandle.getTotalResults());
+
+        QueryManager queryManager = runAsTestUserWithRoles("data-hub-common").getFinalClient().newQueryManager();
+        assertThrows(ForbiddenUserException.class, ()->queryManager.search(queryManager.newRawQueryByExampleDefinition(rawHandle), new SearchHandle()));
```

Comment:
Same thing here - use an assertion message to explain why this doesn't work. A benefit of the message is that when someone searches on "eval-search-string" in the codebase, they'll find both the role that inherits it and the test that explains why it's inherited